HighLevelTestCases/TestCasesAPPL.md

```diff
@@ -0,0 +1,12 @@
+# To są przypadki testowe dla COSMIC applet. 
```

Comment:
Please remove "to są" as it sounds unprofessinal

Reply:
**YES**